### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by johndunlap** [glitchwrks/xt_ide#2]
I'm trying to build a hardcard for my IBM 5150 using XT-IDE as the controller. There are two features which would make this considerably easier(nicer) which I hope would be easy to add. 
1) 2 extra screw holes on the side opposite the existing mounting bracket. This would allow the board to be attached to a metal back plate with 4 screws instead of 2.
2) Take 12 volts off the ISA bus at B9 and add a right angle 4 pin header or mini-molex with the following pinout:

Pin | Name | Color | Description
-- | -- | -- | --
1 | +5V | Red | +5 VDC
2 | GND | Black | +5 V Ground
3 | GND | Black | +12 V Ground (Same as +5 V Ground)
4 | +12V | Yellow | +12 VDC

This would allow a 3.5" IDE spinner to be powered directly off the XT-IDE without the need for a molex connector from the power supply.

**Comment by johndunlap** [comment]
With regard to #2, this feature already exists on the Monotech XT-IDE card: https://monotech.fwscart.com/XT-IDE_Bootable_8-bit_ISA_IDE_Interface_Card/p6083514_19478732.aspx

**Comment by chapmajs** [comment]
Hi there, thanks for your interest in the project! The power plug was actually deleted off of the XT-IDE rev 3 boards when I reworked the rev 2 design because we determined very few people needed it. If you're wanting a hardcard workalike device, I'd recommend going with an industrial Flash module (DOM, IDE SSD). They draw significantly less power and don't put the lever strain on the ISA slot/bracket that a mechanical hardcard setup would.

I would caution against buying Monotech stuff, I've had several folks buy XT-IDEs from me after buying some of the knockoffs and having them fail after some time. That Monotech board is basically our open source XT-IDE design, just with our copyright information removed. The proliferation of these cut rate knockoffs are actually why I've stopped selling the rev 4 boards, kits, and assembled cards.

**Comment by johndunlap** [comment]
I currently have a Glitchworks rev 4 XT-IDE with a 256m DOM(which I assembled from a kit). It works perfectly! I completely agree that it is difficult to beat the functionality of a DOM for the price. However, for me, being able to hear the hard drive working is part of the nostalgia of having a vintage PC. I could put a spinner into one of the drive bays but I'm trying to recreate the form factor of my first computer, which was an IBM 5150 with a 20m Quantum Hardcard and two 360k floppy drives.

As this is a feature that very few people need, it might make sense for me to add it back myself in KiCAD and having a custom board made.

**Comment by chapmajs** [comment]
Absolutely! That's one of the major points of keeping this an open source community based projects: people can hack what they need onto the board if it's not something that makes sense to add to the core design! On the Vintage Computer Forums, we'd talked briefly about doing a full length ISA version with mounting holes in the FR4 circuit board for laptop sized IDE drives, though that probably won't get you the mechanical feel you want.

Feel free to fork the repository and make your changes, I'll link to them from the README and the XT-IDE page on glitchwrks.com once you've got it squared away! If you need any help on your work, don't hesitate to ask either here, or better yet ask on the VC Forums http://www.vcfed.org/forum/ which is where most of the development for these projects is discussed.

The only thing about doing a small run of boards is that to have the hard gold edge plating done properly, it'll be very expensive. You can get away with ENIG if you don't mind with having to clean the edge connector periodically -- ENIG is such a thin layer of gold (a few atoms thick) that you'll basically immediately be down to the nickel.

**Comment by johndunlap** [comment]
I have a small CNC machine. Would it be possible to mill a single board or are there too many layers?

**Comment by chapmajs** [comment]
I wouldn't bother trying to mill a board, it's only two layer but you'd be stuck having to either do through-hole plating at home (involved process) or soldering wires through all the vias and soldering the ICs on top and bottom (not fun). Getting a circuit board made in small quantity isn't expensive anymore, even for largeish ones, but getting one made with actual selective hard gold plating *is* still expensive.

Selective hard gold plating is the proper surface finish for edge connectors, like the ISA card edge. The wear resistance and contact life is far superior to ENIG or HASL (HASL being inappropriate for anything but prototypes). It's a two step electroplating process and most of the cheap Chinese board houses either aren't set up for it at all, or aren't set up to do it properly. You'll notice that none of the knockoff boards are using actual selective hard gold plating...

Anyway, the production XT-IDE rev 3 and 4 boards are run at PCB Cart, which is a Chinese board shop that does actually do real selective hard gold plating. They will probably be your cheapest bet if you're wanting real hard gold plating; if it's not that important from you, use pcbshopper.com and order ENIG (every one of them should be able to do ENIG for a good price).

**Comment by johndunlap** [comment]
"On the Vintage Computer Forums, we'd talked briefly about doing a full length ISA version with mounting holes in the FR4 circuit board for laptop sized IDE drives"

That's actually a really cool idea!
